src/components/post/index.js

```diff
@@ -33,6 +46,14 @@ const Post = ({postId, name, date, content, comments = [], likes = 0, refreshPos
     setIsComment(!isComment)
   }
 
+  useEffect(() => {
+    const newDate = new Date(date)
+    const day = newDate.getDate()
+    const month = newDate.toLocaleString('eng', { month: 'long' })
```

Comment:
I don't think `'eng'` is a correct locale code.

Take a look at this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString